saleor/graphql/order/filters.py

```diff
@@ -182,6 +183,12 @@ def filter_by_order_number(qs, _, values):
     return qs.filter(number__in=values)
 
 
+def filter_by_checkout_tokens(qs, _, values):
+    if not values:
+        return qs
+    return qs.filter(checkout_token__in=values)
```

Comment:
Using `checkout_token` in filters requires a proper index. Right now on DB with 4.5 mln orders the execution time can be counted in seconds.
![image](https://github.com/user-attachments/assets/245185b4-5765-416e-ac2f-b2b318283606)

Reply:
I will add an index and test it

Reply:
Tested on a 5 mln order set, now it should work in less than 0.2 seconds